.travis.yml

```diff
@@ -7,16 +7,23 @@ php:
   - 7.0
 
 env:
-  - TEST_SUITE=marketplate_eqp
-  - TEST_SUITE=ecg_coding_standards
+  global:
+    - COMPOSER_BIN_DIR=~/bin
+  matrix:
+    - CODING_STANDARD=Ecg SEVERITY=5
+    - CODING_STANDARD=MEQP1 SEVERITY=5
+    - CODING_STANDARD=MEQP1 SEVERITY=9
```

Comment:
Might as well be 10 since there isn't any MEQP1 tests with severity 9.
Chose 9 because is the lowest value that is passing.